software/helper scripts/updater.sh

```diff
@@ -1,12 +1,85 @@
 #!/bin/bash
-fbv -d 1 /update.png
+
+#--------------------
+# YIO updater script
+#--------------------
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root (probably not a good idea to run this script yourself)." 1>&2
+   exit 1
+fi
+
+if [ -z $1 ]; then
+    echo "This script requires the URL to download (probably not a good idea to run this script yourself)."
+    exit 1
+fi
+
+
+#--------------------
+# note: Since OSX 10.11, Apple reinforces System Integrity Protection and locks down:
+# - /System
+# - /sbin
+# - /usr (with the exception of /usr/local subdirectory)
+#
+# Since the developers of the yio-remote use macs, i adopted this script to work with the /usr/local/yio-remote target path.
+# That is also why we require root access
+#--------------------
+
+
+#--------------------
+# TODO: Rework based on path?
+#--------------------
+fbv -d 1 /usr/local/yio-remote/update.png
```

Comment:
Same as above, if we only run it on the pi

Reply:
yeah but currently it did `fbv -d 1 /update.png` (it was missing a path); can it be then this: `fbv -d 1 /usr/bin/yio-remote/update.png` ?

Reply:
edit: didn't see your change below; this is resolved.